### PR TITLE
Books Library Project

### PR DESCRIPTION
**Pull request by necmigunduz**
In this project, we did;
-create a Book Library project by using JS, HTML5, and CSS3;
-create add, remove, and change read status functions to the project.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello Team 👋

**Nice work on this. Well written readme, and nice idea adding persistence with localStorage. 👌**

However, let's make a couple of changes.

- [ ] Kindly deploy the project and include the live deploy link in the readme.

- Please refer to other comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: assets/script.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
+let library = [];
+const container = document.querySelector('.container');
+const title = document.querySelector('#title');
+const author = document.querySelector('#author');
+const pages = document.querySelector('#pages');
+const read = document.querySelector('#read');
+// const notRead = document.querySelector('#not-read');
+// const read = document.querySelectorAll('input[name='read-check']');
```
Comment (mikenath223) [inline assets/script.js]: - [ ] Please remove these commented code lines if you need to refer to the previous versions of the code, consider using git history.

**Comment by mikenath223** [comment]
- [ ] OPTIONAL - Consider adding form validation so the user won't be able to create an empty book.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :construction:

### Hi @necmigunduz & @wandji20  :wave: 
Good job with the changes! :ok_hand: We are almost there.
However, we do have some changes to carry out before we proceed (see comments below).

**[ REQUIRED CHANGES ]** 

- [x] As mentioned by the last reviewer, kindly deploy your project and provide the link on your Readme.
- [x] The comments in `~/assets/script.js` are still there. Kindly remove them.



* * *
**When you have made the specified changes, please submit another code review. Thank you!**

**Please drop your comments on the PR so that either previous or next reviewers can reply.**

**If you feel like you need help or you have any question(s), I'm @taofeekolalere on slack**

* * * 
Regards,
**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Comment by necmigunduz** [comment]
The app is deployed to Heroku and the live link is in the Readme now. The comments are cleaned.

**Comment by teekaytech** [comment]
> ## Status: Changes Required warning construction
> ### Hi @necmigunduz & @wandji20 wave
> 
> Good job with the changes! ok_hand We are almost there.
> However, we do have some changes to carry out before we proceed (see comments below).
> 
> **[ REQUIRED CHANGES ]**
> 
>     * [x]  As mentioned by the last reviewer, kindly deploy your project and provide the link on your Readme.
> 
>     * [x]  The comments in `~/assets/script.js` are still there. Kindly remove them.
> 
> 
> **When you have made the specified changes, please submit another code review. Thank you!**
> 
> **Please drop your comments on the PR so that either previous or next reviewers can reply.**
> 
> **If you feel like you need help or you have any question(s), I'm @taofeekolalere on slack**
> 
> Regards,
> **Taofeek Olalere [Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
> **Happy Coding smiley**

# Status: Approved :green_circle: 

### Congratulations @necmigunduz & @wandji20 :tada: :clinking_glasses: 

**This project has passed all the mandatory requirements :clap: 
It is therefore approved :heavy_check_mark:**

**Comment(s)**
- Since the project has been approved earlier, you're all good to go.
- You can use GitHub markdown to format your live demo link. i.e. `[Live demo text](live demo URL)`


Again, congratulations, you can merge to the default branch.
Do not forget to fill the project completion form, as this is important for your weekly progress score.

**_Good luck with your next task._**

* * * 
**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Pull request merged**